src/app/campaign/creative/creative-card.component.ts

```diff
@@ -21,28 +22,18 @@ import { Creative } from '../store/models';
   styleUrls: ['./creative-card.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class CreativeCardComponent implements OnInit, OnDestroy {
+export class CreativeCardComponent implements OnInit {
   @Input() creative: Creative;
   @Input() creativeLink: string;
   creativeForm: FormGroup;
-  disableChangedSubscription: Subscription;
 
   ngOnInit() {
     this.creativeForm = this.formContainer.control as FormGroup;
-    this.disableChangedSubscription = this.creativeForm.get('enabled').valueChanges.subscribe(enabled => this.setDisabledState(!enabled));
```

Comment:
Ah, okay, so this isn't needed now that the form doesn't "react" to the toggle changing.  (You don't actively disable the other field).